backend/service/adminServiceDelete.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const AWS = require('aws-sdk');
+const db = new AWS.DynamoDB.DocumentClient();
+const uuid = require('uuid');
+const sUtils = require('../utils/service-utils');
+const utils = require('../utils/utils');
+require('dotenv').config();
+
+async function adminServiceDelete(event, context, callback) {
+    try {
+        const request = event.arguments;
+
+        await db.delete({
+            TableName: process.env.SERVICE_TABLE,
+            Key: {
+                id: request.serviceId
+            },
+            ConditionExpression: 'attribute_exists(id)'
+        }).promise().catch(err => {
```

Comment:
Using the catch here will not stop the processing of the function. Please, remove it. 
The catch from try/catch will handle the failures from try block.